inst/qml/Prophet.qml

```diff
@@ -91,28 +91,25 @@ Form
 		{
 			value: "logistic"
 			label: qsTr("Logistic")
-		}
-	}
-
-	Group
-	{
-		DoubleField
-		{
-			name: "constantCapacity"
-			label: qsTr("Constant carrying capacity")
-			enabled: cap.count === 0
-			visible: growth.value === "logistic"
-			negativeValues: true
-			fieldWidth: 100
-		}
-		DoubleField
-		{
-			name: "constantMinimum"
-			label: qsTr("Constant saturating minimum")
-			enabled: floor.count === 0
-			visible: growth.value === "logistic"
-			negativeValues: true
-			fieldWidth: 100
+			
+			DoubleField
+			{
+				name: "constantCapacity"
+				label: qsTr("Constant carrying capacity")
+				enabled: cap.count === 0
+				visible: growth.value === "logistic"
+				negativeValues: true
+				fieldWidth: 100
+			}
+			DoubleField
+			{
+				name: "constantMinimum"
+				label: qsTr("Constant saturating minimum")
+				enabled: floor.count === 0
+				visible: growth.value === "logistic"
+				negativeValues: true
+				fieldWidth: 100
+			}
```

Comment:
I am also wondering whether we could drop the word "constant"; I think if we show "Carrying capacity" and "Saturating minimum" with an option to specify a fixed value, it's pretty much obvious that it's a constant.

Reply:
I dropped the "Constant", aligned the two double boxes, and made them smaller. I chose `childrenOnSameRow: true` because then it looks a bit nicer I think. @Kucharssim do you agree?

![image](https://user-images.githubusercontent.com/63114869/130782922-ce9d79da-a709-4495-912b-0f83aa1554c0.png)

Reply:
I disagree, I like it put under the option...